index.html

```diff
@@ -1,29 +1,30 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Home page</title>
-    <link rel="stylesheet" href="styles.css" />
-  </head>
-  <body>
-    <img src="img/seven-o-clock.svg" alt="Animated clock logo" class="logo" />
-    <section class="head-line">
-      <header class="tool-bar">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+     <link rel="stylesheet" href="styles.css"> 
+     
+</head>
+<body>
+  
+  <section class="head-line">
+    <header class="tool-bar" id="navbar-js">
         <div class="toolbar-dark-compact">
-          <a class="john-doe" href="https://www.w3schools.com">Yasen</a>
-          <nav class="normal-button">
-            <img class="humberger-img" src="img/Union.png" alt="Mobile menu" />
-            <ul class="drop-content">
-              <li><a href="">Portfolio</a></li>
-              <li><a href="">About</a></li>
-              <li><a href="">Contact</a></li>
-              <li>
-                <a href=""><img src="img/envelop.svg" alt="Email" /></a>
-              </li>
-            </ul>
-          </nav>
+            <a class="john-doe" href="https://www.w3schools.com">Yasen</a>
+            <nav class="normal-button" >  
+                <img  id="humberger-icon" class="humberger-img" src="img/Union.png" alt="">
```

Comment:
- [ ] Currently, your close button is not showing. Kindly add a (`**dot.**`) to the relative path of the close icon image so that it can be visible e.g 

Replace your existing path with this 👇 
` <img src="./img/x-lg.svg" class="close-tap-img">`